src/main/java/com/digitax/controller/AuthController.java

```diff
@@ -77,30 +79,51 @@
 				.collect(Collectors.toList());
 		
 		Object Sessionobj = new SessionResponse(jwt,jwtExpirationMs);
-
-	    JwtResponse obj = new JwtResponse(Sessionobj,userDetails);
-		 return new ResponseEntity<>(ApiRes.success(obj,200).setMessage("Success."), HttpStatus.OK);
+		JSONObject userDetailsObj=new JSONObject();    
+		userDetailsObj.put("id",userDetails.getId());  
+		userDetailsObj.put("username",userDetails.getUsername());
+		userDetailsObj.put("email",userDetails.getEmail());
+		userDetailsObj.put("authorities",userDetails.getAuthorities());
+
+	    JwtResponse obj = new JwtResponse(Sessionobj,userDetailsObj);
+	    JSONObject statusObj=new JSONObject();    
+		statusObj.put("status_code",200);  
+		statusObj.put("message","SUCCESS");
+		 return new ResponseEntity<>(ApiRes.success(obj,statusObj), HttpStatus.OK);
 		 } catch (Exception ex) {
 			 
 			 //logger.error("Unauthorized user.");
-             return ResponseEntity.status(HttpStatus.OK).body(ApiRes.fail(401).setMessage("Unauthorized user."));
+			 JSONObject statusObj=new JSONObject();    
+				statusObj.put("status_code",401);  
+				statusObj.put("message","Unauthorized user.");
+             return ResponseEntity.status(HttpStatus.OK).body(ApiRes.fail(statusObj));
 	        }
 	}
 
+	@SuppressWarnings({ "unchecked", "deprecation" })
 	@PostMapping("/signup")
 	public ResponseEntity<?> registerUser(@Valid @RequestBody SignupRequest signUpRequest) {
 		if (userRepository.existsByUsername(signUpRequest.getUsername())) {
-			return ResponseEntity.status(HttpStatus.OK).body(ApiRes.fail(400).setMessage("Username is already taken!"));
+			JSONObject statusObj=new JSONObject();    
+			statusObj.put("status_code",new Integer(400));  
+			statusObj.put("message","Username is already taken!");
+			return ResponseEntity.status(HttpStatus.OK).body(ApiRes.fail(statusObj));
 			
 		}
 
 		if (userRepository.existsByEmail(signUpRequest.getEmail())) {
-			return ResponseEntity.status(HttpStatus.OK).body(ApiRes.fail(400).setMessage("Email is already in use!"));
+			JSONObject statusObj=new JSONObject();    
+			statusObj.put("status_code",new Integer(400));  
```

Comment:
@Jayanta-Redapple Add these keys as Constants. This way there is one point of reference, and reduces chance of misspellings. Avoid using hardcoded values at all times.